docs/cloudlinux-os-plus/README.md

```diff
@@ -1528,16 +1540,78 @@ To disable both suites:
 cloudlinux-awp-admin set-suite --suite=accelerate_wp,accelerate_wp_premium --disallowed --users=<username>
 ```
 
+### WHMCS billing
+
+CloudLinux developed it's own WHMCS plugin 
+which provides you AccelerateWP billing integration our of the box. 
+Check out [the documentation](WHMCS_DOCS_URL_HERE) to find how to install and use the plugin.
+
+### Other billing integration
+
+As AccelerateWP Premium is the feature that works on subscription basis, 
+we made it possible for hosters to integrate their existing billing systems with our plugin and sell the 
+feature for their users.
+
+When AccelerateWP Premium is enabled in admin interface, users get proposal to upgrade their subscription.
+
+![](./images/awp/AWPUpgradeNoLink.png)
+
+When user upgrades the subscription to the plan with AccelerateWP support, 
+billing must execute the following command on the server:
+```
+cloudlinux-awp-admin set-suite --suite=accelerate_wp_premium --allowed --users=<username>
+```
+
+This command makes two things:
+- allow user to install plugin, bypassing the upgrade window;
+- reports premium as being used to CLN, starting the billing cycle for hoster.
+
+
+When user terminates or downgrades plan, the following command must be exetured:
```

Comment:
executed?

Reply:
In the way it is currently written, it is not explicit who should execute the command below. Should it be the hoster, or it happens automatically?

Reply:
Done